geode-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -192,43 +244,50 @@ public void channelInactive(ChannelHandlerContext ctx) {
     if (logger.isDebugEnabled()) {
       logger.debug("GeodeRedisServer-Connection closing with " + ctx.channel().remoteAddress());
     }
+    commandQueue.offer(TERMINATE_COMMAND);
```

Comment:
I think we should review the commandQueue code and see if we might continue to process commands in the queue after this happens. For example this code adds TERMINATE_COMMAND to the end of the queue so do we end up still processing any commands already queued? We should also check that the "quit" command does not allow commands queued after it to still be processed on the server

Reply:
This will be addressed in a separate PR.